gui/wxpython/lmgr/frame.py

```diff
@@ -1141,7 +1148,12 @@ def OnChangeMapset(self, event):
                 GError(parent=self,
                        message=_("No mapset provided. Operation canceled."))
                 return
-            self.ChangeMapset(mapset)
+            if can_switch_mapset_interactively(self,
+                                               grass.gisenv()['GISDBASE'],
+                                               grass.gisenv()['LOCATION_NAME'],
```

Comment:
This is actually two module calls. Do `gisenv = grass.gisenv()` and then use it like elsewhere.